api/ReadableStreamDefaultController.json

```diff
@@ -11,6 +11,10 @@
           "chrome_android": {
             "version_added": "89"
           },
+          "deno": {
+            "version_added": "1.0",
+            "notes": "This interface was not exposed on the global scope until Deno 1.7."
```

Comment:
Another case where it'll be good to have two statements, one partial_implementation